scripts/configure-installer-tests-cluster.sh

```diff
@@ -15,9 +15,9 @@ OC_STABLE_LOGIN="false"
 CI_OPERATOR_HUB_PROJECT="ci-operator-hub-project"
 # Copy kubeconfig to temporary kubeconfig file
 # Read, Write and Execute permission to temporary kubeconfig file
-mkdir -p ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
+mkdir -p ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp
 cp ${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
-chmod 764 ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
+chmod 644 ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
```

Comment:
Let's see how it goes. 

I still doubt this will pass though as you might need to give write access to the group as openshift uses randomuid and hence permission on gid will be more important. This also means group owner of file will  have to be root.

Again lets wait for CI